### ISSUE TITLE
MM or M in fun with LoKi functors

### ISSUE THREAD

**Issue by saschastahl** [lhcb/first-analysis-steps#181]
Hi all,
I think this lesson should not mention MM. It should use M. When the OfflineVertexFitter was used M and MM were the same. Since we use the LoKi::VertexFitter they are slightly different. MM is a combination of the daughter 4-momenta, while M is the result of the vertex fit and should be slightly better than MM.

**Comment by alexpearce** [comment]
I agree that `M` is ‘better’, and its what analysts should almost always use. Still, part of me wants to keep a mention of `M` in the lesson (or some lesson) as students invariably asked what the difference is.

Maybe the best solution is to add a callout box somewhere, explaining the difference and saying which one should be used.

**Comment by betatim** [comment]
Some discussion of this (and work done to fix it): #148

**Comment by saschastahl** [comment]
I knew this was discussed before :). My reaction was triggered by seeing _MM in a presentation. A callout box might be the best solution.

**Comment by betatim** [comment]
[To use the search box, you have.](https://github.com/lhcb/first-analysis-steps/search?utf8=%E2%9C%93&q=MM)

**Comment by saschastahl** [comment]
Not sure we want to replace mm with m :)
